core/src/test/java/org/kohsuke/stapler/DataBindingTest.java

```diff
@@ -412,4 +414,34 @@ public void testDerivedProperty() {
         DerivedProperty r = bind("{items:[1,3,5]}",DerivedProperty.class);
         assertEquals(Arrays.asList(1,3,5),r.getItems());
     }
+
+    public static class FluentSetter {
+        private List<String> items;
+
+        @DataBoundConstructor
+        public FluentSetter() {}
+
+        /**
+         * WIP This tests passes in stapler, but fluent doesn't work in jenkins...
+         * @see FluentPropertyBeanIntrospector
+         *
+         * It is also possible to transform it to `withXXX()` to have clear vision.
+         */
+        @DataBoundSetter
+//        public void setItems(List<String> items) {
+        public FluentSetter setItems(List<String> items) {
```

Comment:
Seems potentially useful, but why is it filed here? Should be split into a separate PR.

Reply:
Fluentsetters was exactly the reason to update beanutils that provides fluentbinder used in core... But i had working testcase here and failing in core + unclear test failure stuck me.